examples/apis/example.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name" : "Example",
+    "slug" : "example",
```

Comment:
what do you think about renaming `slug` to `id` to make it more explicit, as this field is real identification for auth/route now?